src/App.php

```diff
@@ -126,6 +126,51 @@ class App
 	 */
 	private $pConfig;
 
+	/**
+	 * Set the user ID
+	 *
+	 * @param int $user_id
+	 * @return void
+	 */
+	public function setUserId(int $user_id)
```

Comment:
This should be logically changed to the following
```suggestion
	public function setLoggedInUserId(int $user_id)
```

Reply:
Done